packages/casting/src/types.js

```diff
@@ -40,14 +40,18 @@ export {};
  */
 
 /**
- * @see {ChangeFollower}
  * @template T
```

Comment:
The base no longer needs to be a template.
```suggestion
```

Reply:
Ah good catch, removed